src/main/preload.ts

```diff
@@ -173,13 +181,41 @@ const electronHandler = {
     ipcRenderer.invoke('setSelectedChallongeTournament', slug),
   getChallongeTournament: (slug: string): Promise<void> =>
     ipcRenderer.invoke('getChallongeTournament', slug),
+  getCurrentParryggTournaments: (): Promise<Map<string, ParryggTournament>> =>
+    ipcRenderer.invoke('getCurrentParryggTournaments'),
+  getSelectedParryggTournament: (): Promise<ParryggTournament | undefined> =>
+    ipcRenderer.invoke('getSelectedParryggTournament'),
+  setSelectedParryggTournament: (slug: string): Promise<void> =>
+    ipcRenderer.invoke('setSelectedParryggTournament', slug),
+  setSelectedParryggSetChain: (setChain: ParryggSetChain): Promise<void> =>
+    ipcRenderer.invoke('setSelectedParryggSetChain', setChain),
+  getSelectedParryggSet: (): Promise<ParryggSet | undefined> =>
+    ipcRenderer.invoke('getSelectedParryggSet'),
+  setSelectedParryggSetId: (setId: string): Promise<void> =>
+    ipcRenderer.invoke('setSelectedParryggSetId', setId),
+  getParryggTournament: (slug: string, recursive?: boolean): Promise<void> =>
+    ipcRenderer.invoke('getParryggTournament', slug, recursive),
+  getParryggEvent: (id: string): Promise<void> =>
+    ipcRenderer.invoke('getParryggEvent', id),
+  getParryggPhase: (id: string): Promise<void> =>
+    ipcRenderer.invoke('getParryggPhase', id),
+  getParryggBracket: (id: string): Promise<void> =>
+    ipcRenderer.invoke('getParryggBracket', id),
```

Comment:
don't interleave with challonge methods, keep them in contiguous blocks as much as possible